parm/archive/gdas_restarta.yaml.j2

```diff
@@ -0,0 +1,49 @@
+gdas_restarta:
+    {% set head = "gdas.t" + cycle_HH + "z." %}
+    name: "GDAS_RESTARTA"
+    target: "{{ ATARDIR }}/{{ cycle_YMDH }}/gdas_restarta.tar"
+    required:
+        {% if current_cycle != SDATE and MODE == "cycled" %}
+        - "{{ COM_ATMOS_ANALYSIS | relpath(ROTDIR) }}/{{ head }}atminc.nc"
+        {% for iauhr in IAUHRS if iauhr != 6 %}
```

Comment:
These should probably be IAUFHRS, not IAUHRS.

Reply:
Thanks for this catch.  I'll fix this and regen the archives.